source/solvers/postprocessing_velocities.cc

```diff
@@ -12,26 +12,37 @@ AverageVelocities<dim, VectorType, DofsType>::calculate_average_velocities(
   const Parameters::PostProcessing &post_processing,
   const double &                    current_time,
   const double &                    time_step,
+  const bool &                      is_output_iteration,
```

Comment:
The idea here I guess is to only calculate average when you really need it right? Why not calculate it then when we ask for it instead? (in the getter) this would prevent this argument being passed here. To me it is weird to have a calculation routine depending on an i/o argument.

Reply:
My idea was to only calculate the average_velocities = sum__velocities_dt / dt when we want the data out or if reynolds stress is enable (because it needs the average velocities at each iteration). If calculate_reynolds_stress is removed, average_velocities should be calculated everytime so no needs to have an boolean of something in the getter to calculate it anymore.